packages/devextreme/js/core/component.js

```diff
@@ -82,6 +86,11 @@ export const Component = Class.inherit({
         this._disposingCallbacks = _disposingCallbacks || Callbacks();
         this.postponedOperations = new PostponedOperations();
         this._createOptions(options);
+
+        if(!isLicenseVerified) {
+            isLicenseVerified = true;
+            verifyLicense(Config().license, version);
```

Comment:
[copypaste]
I suggest u totally get rid of parameter passing here cause is does not depend from the component state